lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] if entity.corporation? && !entity.minor?
```

Comment:
You don't need to also check for `!entity.minor?` that will always be false if `entity.corporation?` is true.
```suggestion
            return [] if entity.corporation?
```